emulator/blockchain.go

```diff
@@ -555,8 +567,12 @@ func configureFVM(blockchain *Blockchain, conf config, blocks *blocks) (*fvm.Vir
 		CapabilityControllersEnabled: true,
 		CoverageReport:               conf.CoverageReport,
 	}
+	rt := conf.runtime
+	if conf.runtime == nil {
+		rt = runtime.NewInterpreterRuntime(config)
```

Comment:
Unrelated: maybe rename `config` to `runtimeConfig` – it's a bit confusing that there's both a `conf` and a `config` in the function

Reply:
Good point :+1: 
Fixed in https://github.com/onflow/flow-emulator/pull/476/commits/a349e2bb4826f44a787ee4f514f3023e65c42799